### ISSUE TITLE
specify different pattern for positive and negative number

### ISSUE THREAD

**Issue by titimoby** [foretagsplatsen/numbro#139]
Is there a way to specify a format pattern that is different if number is positive or negative.
Something coming from java world:
`numbro(-15.2).format('###.00;(###.##)')`
which returns:
(15.2)
and:
`numbro(-15.2).format('###.00;(###.##)')`
which returns:
`15.20`

**Comment by BenjaminVanRyseghem** [comment]
I do not think this is supported yet, but could be easily (I think) implemented, by providing an option map to `format` instead of just a string, and use it depending on the number value

**Comment by titimoby** [comment]
I think for the moment, it's possible to just use:
`numbro(-15.2).format('(###.##)')`
and forget the capacity to defined a really different format between positive and negative.
Thanks for the answer
